src/problems/n_puzzle/n_puzzle_problem.py

```diff
@@ -0,0 +1,58 @@
+from base import Problem
+from problems.n_puzzle.n_puzzle_state import NPuzzleState
```

Comment:
Jako że macie to w `__init__.py` 
![image](https://user-images.githubusercontent.com/21079319/134075051-5e81b761-5efd-4706-a82e-37a2c256082a.png)

można skrócić do 
`from problem.n_puzzle import NPuzzleState`